src/webview_provider/altimateWebviewProvider.ts

```diff
@@ -0,0 +1,258 @@
+    const { command, syncRequestId, ...params } = message;
+
+    try {
+      switch (command) {
+        case "openURL":
+          if (!params.url) {
+            return;
+          }
+          env.openExternal(Uri.parse(params.url as string));
+          break;
+        case "datapilot:toggle":
+          if (params.open) {
+            await commands.executeCommand(
+              "dbtPowerUser.datapilot-webview.focus",
+            );
+          }
+          break;
+        case "datapilot:message":
+          await commands.executeCommand("dbtPowerUser.datapilot-webview.focus");
+          // Adding timeout to let the datapilot complete rendering
+          setTimeout(() => {
+            sharedStateManager.postMessage(message);
+          }, 500);
+          break;
+        case "updateConfig":
+          console.log(
+            "Updating config",
+            (params as UpdateConfigProps).key,
+            (params as UpdateConfigProps).value,
+          );
+          // If config is for preview feature, then check keys
+          const shouldUpdate =
+            !(params as UpdateConfigProps).isPreviewFeature ||
+            this.altimateRequest.handlePreviewFeatures();
+          if (shouldUpdate) {
+            await workspace
+              .getConfiguration("dbt")
+              .update(
+                (params as UpdateConfigProps).key,
+                (params as UpdateConfigProps).value,
+              );
+          }
+          if (syncRequestId) {
+            this._panel!.webview.postMessage({
+              command: "response",
+              args: {
+                syncRequestId,
+                body: {
+                  updated: shouldUpdate,
+                },
+                status: true,
+              },
+            });
+          }
+          break;
+        default:
+          break;
+      }
+    } catch (err) {
+      console.error("error while handling command", err);
+    }
+  }
+
+  resolveWebviewView(
+    panel: WebviewView,
+    context: WebviewViewResolveContext<unknown>,
+    _token: CancellationToken,
+  ): void | Thenable<void> {
+    console.log("AltimateWebviewProvider:resolveWebviewView -> ");
+    this._panel = panel;
+    this.setupWebviewOptions(context);
+    this.renderWebviewView(context);
+    setTimeout(() => {
+      this.initWebView();
+    }, 1000);
+  }
+
+  changedActiveColorTheme() {
+    if (!this._panel) {
+      return;
+    }
+
+    this._panel.webview.postMessage({
+      command: "setTheme",
+      args: { theme: this.getTheme() },
+    });
+  }
+
+  private initWebView() {
+    this.changedActiveColorTheme();
+  }
+
+  private setupWebviewOptions(context: WebviewViewResolveContext) {
+    this._panel!.description = this.panelDescription;
+    this._panel!.webview.options = <WebviewOptions>{
+      enableScripts: true,
+      localResourceRoots: [
+        Uri.file(
+          path.join(
+            this.dbtProjectContainer.extensionPath,
```

Comment:
Fishy, should use uri.fesPath or won't work on windows.

Reply:
ok, i ll try with uri.fsPath